src/main/java/com/adyen/model/terminal/TerminalAPIRequest.java

```diff
@@ -73,4 +72,19 @@ public String toString() {
                 "saleToPOIRequest=" + saleToPOIRequest +
                 '}';
     }
+
+    /**
+     * Checks if SaleToAcquirerData is null
+     *
+     * @return true if sale to acquirer data is null
+     */
+    public Boolean isSaleToAcquirerDataModelSet() {
```

Comment:
The method returns "true" if there's no SaleToAcquirerData set, so I think the name is misleading.
We could rename it or change it so `SaleToAcquirerData == null` returns false

Reply:
I'd change the order. Didn't see that in my previous review 😬

Reply:
I'd changed the order